api/features/views.py

```diff
@@ -466,30 +467,20 @@ class SimpleFeatureStateViewSet(
     viewsets.GenericViewSet,
 ):
     serializer_class = WritableNestedFeatureStateSerializer
-    permission_classes = [IsAuthenticated, FeatureStatePermissions]
+    permission_classes = [FeatureStatePermissions | MasterAPIKeyFeatureStatePermissions]
```

Comment:
Ah, this probably answers my question from above. Would the following work instead though? 

```python
permission_classes = [[IsAuthenticated, FeatureStatePermissions] | MasterAPIKeyFeatureStatePermissions]
```

I'm not sure how complicated you can get with the permission classes.

Reply:
Nope, we can't, it just has to be a flat list/tuple